src/coreclr/vm/readytoruninfo.cpp

```diff
@@ -802,6 +802,17 @@ ReadyToRunInfo::ReadyToRunInfo(Module * pModule, LoaderAllocator* pLoaderAllocat
         m_nRuntimeFunctions = 0;
     }
 
+    IMAGE_DATA_DIRECTORY * pHotColdMapDir = m_pComposite->FindSection(ReadyToRunSectionType::HotColdMap);
+    if (pHotColdMapDir != NULL)
+    {
+        m_pHotColdMap = (PTR_ULONG)m_pComposite->GetLayout()->GetDirectoryData(pHotColdMapDir);
+        m_nHotColdMap = pHotColdMapDir->Size / sizeof(ULONG);
+    }
+    else
+    {
+        m_nHotColdMap = 0;
```

Comment:
Nit - if we need to zero <code>m_nHotColdMap</code> I guess we should also zero <code>m_pHotColdMap</code> for the sake of consistency. If we zero both in the constructor, we can drop the else block.